CHANGELOG/CHANGELOG-1.x.md

```diff
@@ -16,11 +17,21 @@
 </tr>
 </table>
 
+# v1.1.1
+
+`2021-8-5`
+
+### Bug Fixes
+
+* EDAS shares the same Agent with K8s. ([#1277](https://github.com/erda-project/erda/pull/1277))
+* Releases functions of cloud management platform for free users. ([#810](https://github.com/erda-project/erda-ui/pull/810))
+* Fixed the issue that error occured when editing custom addon of extended service. ([#813](https://github.com/erda-project/erda-ui/pull/813))
```

Comment:
Fix errors caused by editing custom addon of extended service

Reply:
Emmm, these are all translated by documentation engineer.